### PR TITLE
160. Intersection of Two Linked Lists

### PR DESCRIPTION
**Pull request by andreypudov**

### CODE REVIEW

File: problems/core/list.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ListNode:
```
Comment (mletunenko) [inline problems/core/list.py:1]: I don't like this file naming, may be structure.py or data.py

File: problems/helpers/list.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from core.list import ListNode
```
Comment (mletunenko) [inline problems/helpers/list.py:1]: I don't like this file naming, also. viewers/printers?)
I love your idea to organize our extracode in structure)

**Review by mletunenko — APPROVED** [review]

File: problems/core/list.py (hunk 1 above)
Comment (andreypudov) [reply problems/core/list.py:1]: I would like to have all entities separated, all in individual files.

File: problems/helpers/list.py (hunk 2 above)
Comment (andreypudov) [reply problems/helpers/list.py:1]: But this helper is only for the ListNode.

**Pull request merged**